### PR TITLE
Noughts And Crosses

### LINKED ISSUE #4375: [New game]: Noughts And Crosses
### 🎮 Game Request

Noughts and crosses is a game between two players, O and X, who alternate in marking the spaces in a 3×3 board. A player wins by getting three of their own marks in a horizontal, vertical or diagonal row.

### Point down the features

I have multi player game in which players can choose their moves alternatively and player who wins is displayed on the screen.
I have also added a reset score button to reset the game.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by meetarora10**
## PR Description 📜

Noughts and crosses is a game between two players, O and X, who alternate in marking the spaces in a 3×3 board. A player wins by getting three of their own marks in a horizontal, vertical or diagonal row. I have used technologies such as HTML,CSS and JavaScript.
Fixes #4375

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

Screenshot (467)

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @meetarora10 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @meetarora10,
You need to make the following changes:
  - Please add a README.md file inside the newly created folder.
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @meetarora10,
You need to make the following changes:
  - Please add a README.md file inside the newly created folder.
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @meetarora10,
You need to make the following changes:
  - Please modify the main README.md file.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @meetarora10,
You need to make the following changes:
  - Please modify the main README.md file.

Hoping that you will make those changes soon 🚀

**Comment by meetarora10** [comment]
@kunjgit I have made the changes.

**Comment by meetarora10** [comment]
@kunjgit I have resolved the merge conflicts please review once.

File: README.md
Comment (ishita-43) [inline README.md:1]: add your game entry around line number 325 refer other entries for format

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: README.md
Comment (meetarora10) [reply README.md:1]: Done.

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by kunjgit** [comment]
Thank you  @meetarora10 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**